src/Problems/PBO/f_ising_triangle.hpp

```diff
@@ -39,19 +35,27 @@ class Ising_Triangle : public IOHprofiler_problem<int> {
     int i,j,neig;
     int n = x.size();
     int result = 0;
-    int neighbors[6];
+    // int neighbors[6];
+    int neighbors[3];
```

Comment:
Regarding the name of those three Ising model problem, please use `Ising_Ring`, `Ising_Torus`, and `Ising_Triangular`.